config.yml.example

```diff
@@ -26,7 +26,7 @@ auth_file: "/home/git/.ssh/authorized_keys"
 
 # Redis settings used for pushing commit notices to gitlab
 redis:
-  bin: /usr/bin/redis-cli
+  bin: /home/git/gitlab-shell/bin/dummy-redis-cli
```

Comment:
If disabling it isn't something that's exposed by the gitlab-shell scripts (I can't see anything that suggests that it is), then this would seem easier than patching any files that use it, wdyt? Especially if we want to rebase on the latest upstream gitlab-shell